src/Makefile.test_fuzz.include

```diff
@@ -10,7 +10,8 @@ EXTRA_LIBRARIES += \
 TEST_FUZZ_H = \
     test/fuzz/fuzz.h \
     test/fuzz/FuzzedDataProvider.h \
-    test/fuzz/util.h
+    test/fuzz/util.h \
+    test/fuzz/mempool_utils.h
```

Comment:
nit: keep sorted with `m` before `u`?

Reply:
Fixed as of 2f8699a9423c048b1b36a13ea020e687dae245f2